src/pfs/pfs.proto

```diff
@@ -54,6 +54,7 @@ message RepoInfo {
     int64 size_bytes = 1;
   }
   Details details = 7;
+  Project project = 8;
```

Comment:
Since Project is stored inside the Repo Message, it shouldn't also be necessary here